sdk/communication/azure-communication-chat/src/main/java/com/azure/communication/chat/ChatThreadClient.java

```diff
@@ -125,8 +124,7 @@ public void addParticipant(ChatParticipant participant) {
     public Response<AddChatParticipantsResult> addParticipantWithResponse(ChatParticipant participant,
```

Comment:
Should not return anything and raise a ChatError in case of failure (from the review)